cli/src/args.rs

```diff
@@ -44,6 +48,12 @@ pub struct Args {
     /// URL for an HTTP proxy that will be used for all requests if specified
     pub proxy: Option<Url>,
 
+    #[structopt(short = "o", long = "output", default_value = "table")]
+    /// Output format. One of: json, table
+    ///
+    /// Output is provided in table format on stdout by default.
+    pub output: OutputFormat,
```

Comment:
As discussed in slack, we're going to move to a global flag, as we think user readable/data output is a useful paradigm in the vast majority of routes: https://reinfer.slack.com/archives/C0APLNE23/p1625243630379900?thread_ts=1625234925.358800&cid=C0APLNE23

> I think having table/json outputs for the following commands would still be useful:
> - re config add/current/ls/use
>
> The ones I explicitly think it will be never useful in are:
> - re completion
> - re help
>
> which also already don't need args such as --context, --token or --proxy which are other top level args
>
> So I think I am leaning back to global flag